db_plugins/db/mongo/helpers/update_probs.py

```diff
@@ -0,0 +1,93 @@
+from db_plugins.db.mongo.connection import MongoConnection
+from db_plugins.db.mongo import models
+
+"""
+Helper function to create or update the probabilities for an object
+"""
+
+
+def create_of_update_probabilities(
```

Comment:
I imagine it's a typo, but better to correct to "create_or_update_probabilities"

Reply:
ups... fixed